assets/bootstrap_5/build/css/bootstrap_5.css

```diff
@@ -10950,4 +10951,21 @@ a {
 
 .table > :not(:first-child) {
   border-top: 0;
+<<<<<<< HEAD
```

Comment:
Ah, this is not correct, I think.

Reply:
Are you sure ? 🤣

Reply:
i will need to create new PR again for this issue.